zebrium/assets/dashboards/root_cause_as_a_service_sample_dashboard.json

```diff
@@ -0,0 +1,141 @@
+{
+    "author_name": "Datadog",
+    "description": "",
+    "layout_type": "ordered",
+    "template_variables": [],
+    "title": "Zebrium Root Cause as a Service Sample Dashboard",
+    "widgets": [
+        {
+            "definition": {
+                "background_color": "blue",
+                "layout_type": "ordered",
+                "show_title": true,
+                "title": "Zebrium Root Cause as a Service",
```

Comment:
I'd recommend making this title more descriptive about what data the widget is showing

Reply:
We would like to leave this one as-is since it's branding for the Zebrium product